services/retrieve_and_rank/v1.js

```diff
@@ -62,12 +62,33 @@ RetrieveAndRank.prototype.deleteClusters = function(params, callback) {
  * Creates a Solr cluster.
  *
  * @param params An Object representing the parameters for this service call.
- *   This request currently does not require any parameters.
+ *   Optional params
+ *     - clusterName: name to use for identifying the cluster in responses
+ *     - clusterSize: size of the cluster to create
+ *     - clusterOptions: hash of other options used for creating the cluster
  *
  * @param callback The callback.
  */
 RetrieveAndRank.prototype.createCluster = function(params, callback) {
-  return sendRequest('POST', solrClustersPath(), this._options, callback);
+  if (params) {
```

Comment:
I would suggest to use something like:
```
params = params || {};
```
Right now you are not taking into account the scenario when users call `createCluster` without `params`